Tests/MeiliSearchIntegrationTests/SearchTests.swift

```diff
@@ -438,6 +466,34 @@ class SearchTests: XCTestCase {
     self.wait(for: [expectation], timeout: TESTS_TIME_OUT)
   }
 
+  // MARK: Attributes to highlight
+
+  func testSearchPrePostHighlightTags() {
+    let expectation = XCTestExpectation(description: "Search for Books using custom pre and post highlight tags")
+
+    typealias MeiliResult = Result<SearchResult<Book>, Swift.Error>
+    let query = "Joaquim Manuel de Macedo"
+    let attributesToHighlight = ["comment"]
+    let highlightPreTag = "(⊃｡•́‿•̀｡)⊃ "
+    let highlightPostTag = " ⊂(´• ω •`⊂)"
```

Comment:
Asking myself at midnight, what happens if we pass `highlightPostTag = ""`?

Reply:
Nothing happens! It doesnt add any character behind the word.

Reply:
I think this is a design choice that would go against the native API's behavior. Nonetheless maybe we should suggest it in the product. 
Now, for my opinion. I think we should leave the possibility of an empty string. If the user wants to try things out, temporarily create an empty string, create it in a specific process or any other situation. If the user add an empty string and does not understand why no tags are added, I think the first thing he will check is its search parameters